org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/CallHierarchyHandler.java

```diff
@@ -221,13 +249,20 @@ private IMember getCallHierarchyElement(String uri, int line, int character, IPr
 
 		List<CallHierarchyOutgoingCall> result = new ArrayList<>();
 		for (MethodWrapper call : calls) {
+			Collection<CallLocation> callLocations = call.getMethodCall().getCallLocations();
+			if (callLocations != null) {
+				List<Range> ranges = toCallRanges(callLocations);
+				Iterator<CallLocation> iter = callLocations.iterator();
+				while (iter.hasNext()) {
+					iter.next();
```

Comment:
how is that helping since you're not using the current object from the iterator? Might has well check if callLocations is empty in the if block L253.

Reply:
Fixed.